mapbox/src/main/java/org/odk/collect/mapbox/DynamicPolyLineFeature.kt

```diff
@@ -121,8 +119,8 @@ internal class DynamicPolyLineFeature(
             polylineAnnotation = polylineAnnotationManager.create(
                 PolylineAnnotationOptions()
                     .withPoints(points)
-                    .withLineColor(ColorUtils.colorToRgbaString(context.resources.getColor(org.odk.collect.icons.R.color.mapLineColor)))
-                    .withLineWidth(MAPBOX_POLYLINE_STROKE_WIDTH.toDouble())
+                    .withLineColor(lineDescription.getStrokeColor())
+                    .withLineWidth((lineDescription.getStrokeWidth(true)).toDouble())
```

Comment:
I think I'd still prefer the `/ 3` to happen in `mapbox` instead of `LineDescription` needing to know about the specifics of the implementation here. Maybe we could have a `convertStrokeWidth()` helper in `mapbox`?

Reply:
Ok, done.